packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -19,20 +19,36 @@
                 {{ labelText }}
             </form-label>
 
+            <slot v-if="hasInputDescription" />
```

Comment:
Just looking at the storybook entry, this slot content is inside a div – should it be better described in HTML tags for screen readers?

Reply:
At the moment it is in use for user note textarea in f-checkout and the text is basically the same as the one in textarea placeholder which is read by screen readers anyway. But if needed we can always add/change html tags in consuming app/component where we pass the element to the slot.

Reply:
I would at least change it here (and in checkout) so that we use a `p` tag for the text. We shouldn't use block elements on their own like `div`'s to wrap text content as it carries no semantic meaning (plus I'd be interested to know if the placeholder is still read out on screenreaders when some text has been entered in the textarea?).

Unless we think it really limits flexibility, I would probably just take in the text as the slot content (rather than the div and text), and wrap the slot in a `p` tag inside the form field component. That way we aren't reliant on consumers for the accessibility of the component as more people may use this down the line.

Reply:
Makes sense, I will add it to 0.12.1 as well